README.md

```diff
@@ -202,29 +202,38 @@ In the "deploybot" example, we gain a couple benefits from owning the control fl
 
 In building HumanLayer, I've talked to at least 100 SaaS builders (mostly technical founders) looking to make their existing product more agentic. The journey usually goes something like:
 
-1. Decide you want to build an agent
-2. Product design, UX mapping, what problems to solve
-3. Want to move fast, so grab $FRAMEWORK and *get to building*
-4. Get to 80-90% quality bar 
-5a. Realize that 90% isn't good enough for most customer-facing features
-5b. Realize that getting past 90% requires reverse-engineering the framework, prompts, flow, etc
-6. Start over from scratch
-
-**DISCLAIMER**: I'm not sure the exact right place to say this, but here seems as good as any: **this in BY NO MEANS meant to be a dig on either the many frameworks out there, or the pretty dang smart people who work on them**. They enable incredible things and have accelerated the AI ecosystem. 
+<ol>
```

Comment:
This was largely just for formatting the `4a`, `4b` were getting inlined in the preview